tests/test_training.py

```diff
@@ -142,7 +142,7 @@ def test_decentralized_optimizer_step():
 @pytest.mark.forked
 def test_decentralized_optimizer_averaging():
     dht_root = DHT(start=True)
-    initial_peers = [f"127.0.0.1:{dht_root.port}"]
+    initial_peers = dht_root.get_visible_maddrs()
```

Comment:
for subsequent PRs: maybe it's better to paraphrase this as something that clearly states that it is **your** maddrs,

Reply:
I think it's quite clear as is, since the DHT instance usually corresponds to the daemon that is running locally

Reply:
Since we call a method of the `dht_root` object, I think it is clear that these are visible maddrs of `dht_root`.